mdformat/_cli.py

```diff
@@ -23,9 +29,11 @@ def run(cli_args: Sequence[str]) -> int:  # noqa: C901
     try:
         file_paths = resolve_file_paths(args.paths)
     except InvalidPath as e:
-        sys.stderr.write(f'Error: File "{e.path}" does not exist.\n')
-        return 1
+        parser.error(f'File "{e.path}" does not exist.\n')
 
+    # convert args to dict
+    options = dict(args._get_kwargs())
+    options.pop("paths")
```

Comment:
I suggest we remove this and just pass all args to the plugins. The logic is simpler that way.